penn_chime/models.py

```diff
@@ -1,9 +1,118 @@
+from collections import namedtuple
 from typing import Generator, Tuple
 
 import numpy as np
 import pandas as pd
 import streamlit as st
 
+RateLos = namedtuple('RateLos', ('rate', 'length_of_stay'))
```

Comment:
can this be `from defaults import RateLos` instead?

Reply:
Of course, I just noticed that.